capstone/src/main/webapp/js/autocomplete.js

```diff
@@ -120,18 +120,20 @@ function createAutocompleteListElement(collegeName, value) {
 }
 
 /**
- * Adds valid suggestions to list of stored suggestions
- * @param {string} validSuggestion
+ * Keeps track of chosen colleges and the return value is for testing
+ * @param {Array<string, collegeId>} validSuggestion
+ * @return {HTMLDivElement} listElement
  */
 function keepTrackOfChosenColleges(validSuggestion) {
```

Comment:
Now that this function has been refactored, can you update the function name to reflect what it does?